packages/http/src/__tests__/http-prism-instance.spec.ts

```diff
@@ -9,7 +9,7 @@ describe('Http Prism Instance function tests', () => {
   let prism: IPrism<IHttpOperation, IHttpRequest, IHttpResponse, IHttpConfig, { path: string }>;
 
   beforeAll(async () => {
-    prism = createInstance();
+    prism = createInstance({ config: { mock: true } });
```

Comment:
What sort of instance do we have that is not a mock? The pass through proxy stuff that may or may not be implemented yet? 

Maybe we could have a createMockInstance() and createProxyInstance() to avoid some of this nested config object stuff?